h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var maxlength = parseInt(element.getAttribute('data-maxlength'));
+  var counter = element.parentElement.querySelector('.js-character-counter');
+
+  counter.innerHTML = '';
+
+  function updateCounter() {
+    var length = element.value.length;
+
+    counter.innerHTML = length + '/' + maxlength;
```

Comment:
Again, use `textContent` here unless you really want to set HTML and are sure that it is sanitized.